superset/assets/src/chart/chartReducer.js

```diff
@@ -56,6 +56,7 @@ export default function chartReducer(charts = {}, action) {
     [actions.CHART_RENDERING_FAILED](state) {
       return { ...state,
         chartStatus: 'failed',
+        chartStackTrace: action.stackTrace,
```

Comment:
Should `chartStackTrace` be cleared at some point? Otherwise it will be possible to have `chartStatus=rendered` with stale `chartStackTrace`.

Reply:
Good call, even though the logic of what renders is based on `chartStatus`, it's a good idea to not leave expired state in Redux